plugins/reporters/web-app-template/src/components/IssuesTable.jsx

```diff
@@ -56,7 +56,7 @@ const IssuesTable = ({ webAppOrtIssues = [], showExcludesColumn = true }) => {
                         packageName: webAppOrtIssue.package.id,
                         severity: webAppOrtIssue.severity,
                         severityIndex: webAppOrtIssue.severityIndex,
-                        webAppOrtIssue
+                        webAppOrtIssue: webAppOrtIssue
```

Comment:
Should the changes in this file and below have gone to the previous commit?

Reply:
Correct - resolving a merge conflict went wrong - move thes changes now to right place in "Use React state handling" commit.